src/python/pants/engine/process_test.py

```diff
@@ -74,11 +75,20 @@ def test_env(rule_runner: RuleRunner) -> None:
     assert b"VAR2=VAL" in result.stdout
 
 
-def test_output_digest(rule_runner: RuleRunner) -> None:
+@pytest.mark.parametrize("working_directory", ["", "subdir"])
```

Comment:
This is different from #12359 - it tests that the behavior we rely on above (output digests are relative to the working directory) is in fact true.